lms/templates/courseware/courseware-chromeless.html

```diff
@@ -201,4 +201,145 @@
     }
   }());
 </script>
+<%text>
+<script type="text/javascript">
+  /**
+   * @description Overrides AnnotatorJS default resizing behavior with a custom resizer. 
+   *              Automatically intercepts new .annotator-resize elements and applies 
+   *              a more direct, mouse-based resizing for <textarea> elements.
+   */
+
+  (() => {
```

Comment:
This is a self invoking function, but I think the only part you need to run when calling this function is the MutationObserver definition, the class and methods can live outside of this function. @Serafin-dev